src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Ethereum Governance
+description: An introduction to how decisions about Ethereum are made.
+lang: en
+sidebar: true
+---
+
+# Introduction to Ethereum Governance {#introduction}
+
+_If no one owns Ethereum, how are decisions about past and future changes to Ethereum made? Ethereum Governance refers to the process that allows such decisions to be made_
+
+## What is Governance? {#what-is-governance}
+
+The Cambridge dictionary defines governance as:
+
+> _the way that organizations or countries are managed at the highest level, and the systems for doing this_
+
+In essence, it is the systems in place that allows decisions to be made. In a typical organizational structure, the directors' board has the final say in decision-making. In a political system, elected officials advocate for bills that best represent their constituent's desires.
+
+## Decentralized Governance {#decentalized-governance}
+
+No one person owns the Ethereum protocol, but decisions still need to be made about implementing changes best to ensure the longevity and prosperity of the network. This lack of ownership makes traditional organizational governance an incompatible solution.
+
+## Ethereum Governance {#ethereum-governance}
+
+Ethereum Governance is the process by which protocol changes are made. It's important to point out that this process isn't related to how people and applications use the protocol - Ethereum is permissionless. Anyone from anywhere in the world can participate in on-chain activities. There are no rules set for who can or cannot build an application or send a transaction. However, there is a process to propose changes to the core protocol, which these applications run on top of. Since so many people depend on Ethereum's stability, there is a very high coordination threshold for core changes, including social and technical processes, to ensure any changes to Ethereum are secure and acceptable to the community
+
+### Off-chain vs On-chain governance {#off-chain-vs-on-chain}
+
+Ethereum Governance happens off-chain. Off-chain governance means that any protocol change decisions happen through an informal process of social discussion, which, if approved, would be implemented in code.
+
+The opposite approach, on-chain governance, is when proposed protocol changes are decided by a stakeholder vote, usually by holders of a governance token, which happens on the blockchain. With on-chain governance, the proposed protocol changes are already written in code and implemented automatically if the stakeholders approve the changes.
+
+Whilst at a protocol level, Ethereum uses off-chain governance, many use cases built on top of Ethereum, such as DAOs, use on-chain governance.
+
+<ButtonLink to="/dao/">More on DAOs</ButtonLink>
+
+<Divider />
+
+## Who is involved? {#who-is-involved}
```

Comment:
I'd might reorganize this list to sort by what are likely the most common to match with the visitor, e.g. so "ether holders" at the top & "protocol developers" towards the bottom.

Reply:
I think that makes sense. You can go from the outside in